OrderCloud.Integrations.Payment.PayPal/Mappers/PayPalOrderPaymentMapper.cs

```diff
@@ -61,23 +62,25 @@ public List<PurchaseUnit> MapToPurchaseUnit(AuthorizeCCTransaction transaction,
                     }
                 });
             }
-
-            var unit = new PurchaseUnit()
-            {
-                amount = new Amount()
-                {
-                    currency_code = transaction.Currency,
-                    value = transaction.Amount.ToString(CultureInfo.InvariantCulture) ?? null
-                }
-            };
-            if (address != null)
+            else
             {
-                unit.shipping = new Shipping()
+                var unit = new PurchaseUnit()
                 {
-                    address = address
+                    amount = new Amount()
+                    {
+                        currency_code = transaction.Currency,
+                        value = transaction.Amount.ToString(CultureInfo.InvariantCulture) ?? null
```

Comment:
We might need a ?. Here otherwise you could get an error if you try to call toString and the property doesn't exist or is null

Reply:
i tried adding a ? but realized a decimal always has a default value, so it's not necessary